token/program-2022/src/extension/transfer_fee/instruction.rs

```diff
@@ -95,7 +96,10 @@ pub enum TransferFeeInstruction {
     ///   2. `[]` The mint's `withdraw_withheld_authority`'s multisignature owner/delegate.
     ///   3. ..3+M `[signer]` M signer accounts.
     ///   3+M+1. ..3+M+N `[writable]` The source accounts to withdraw from.
-    WithdrawWithheldTokensFromAccounts,
+    WithdrawWithheldTokensFromAccounts {
+        /// Number of token accounts harvested
+        number_of_token_accounts: u8,
```

Comment:
```suggestion
        num_token_accounts: u8,
```
nitty nit; this is just how we usually write such variables